classes/announcement/Collector.inc.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * @file classes/announcement/Collector.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class announcement
+ *
+ * @brief A helper class to configure a Query Builder to get a collection of announcements
+ */
+
+namespace PKP\announcement;
+
+use HookRegistry;
+use Illuminate\Support\Facades\DB;
+use PKP\core\interfaces\CollectorInterface;
+
+class Collector implements CollectorInterface
+{
+    public DAO $dao;
```

Comment:
Should be `\PKP\db\DAO`

Reply:
...or, since you're using this several times in the class, add a `use PKP\db\DAO;` to the top.

Reply:
Ah, correct!